### ISSUE TITLE
Cura Font Rendering went bad as of release 5.2.0

### ISSUE THREAD

**Issue by WardXmodem** [Ultimaker/Cura#13688]
### Application Version

5.2.0 and later

### Platform

Windows 10

### Printer

Anet A8

### Reproduction steps

On Windows 10 with cleartype enabled,  
run any release 5.2.0 or later, observe the readability problems in the options for a slicing.  
Example: "Wall thickness: The two ell's in 'wall' are different thickness; the i is almost unreadable. 
See attachments
![Cura v 5 1 1 OK, V5 2 0 Bad - fonts](https://user-images.githubusercontent.com/6444530/198754244-177193ff-0986-4be0-a894-83abd72f3583.png)

### Actual results

Look at "Wall thickness".  The two ell's in "Wall" aren't the same width; the lower case i is nearly invisible. 

Do the same thing (just open and observe the window) any prior release, they are OK.  I downloaded and installed and tested the following to find at which point the error was introduced: 
------ OK --------
4.13.0 amd64
4.2.1 win64
5.0.0 win64
5.1.1 win64
-------- Show the problem: -------
5.2.0 win64
5.2.1 win64

### Expected results

I expect the readability of the fonts in the options to be good, not bad as they are in 5.2.0, 5.2.1

The issue may be related to Windows ClearType, which uses "sub-pixel" tricks to improve readability of small fonts.  

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Additional information & file uploads

Sigh.  just opening cura causes the font problem.   

[cura.log](https://github.com/Ultimaker/Cura/files/9892557/cura.log)
![Cura v 5 1 1 OK, V5 2 0 Bad - fonts](https://user-images.githubusercontent.com/6444530/198755588-0f3a6e88-b692-41ec-a3ad-7dd1adb0c98b.png)

**Comment by MariMakes** [comment]
Hey @WardXmodem,

Welcome to the Ultimaker Cura Github 🚀 
Thanks for your report 👍

You are not the first to report this, internally we've seen it as well but we've never had solid steps to reproduce it.
Every time we tried to have it appear on purpose we would not see it. I tried again just now, and it looks normal for me. 

I'll bring it up to the team what we can do now you have solid steps to reproduce.
Fingers crossed 🤞

**Comment by MariMakes** [comment]
Hey @WardXmodem,

We've added a ticket to the backlog with the intent to improve this. 
[In our efforts to resolve this issue for Mac we changed this.](https://github.com/Ultimaker/Uranium/blob/c173525a3d994c88502902cbbc9dad50d2b1c040/UM/Qt/qml/UM/Label.qml#L10) 
We'll investigate if this also works for windows. 

For internal reference CURA-9841

Thanks for the report! 👍

**Comment by seltzdesign** [comment]
I am seeing the same issue, but in any version of Cura, currently 5.2.2. I am long time user of Cura and happen to be a UX designer for many years.

<img width="347" alt="image" src="https://user-images.githubusercontent.com/6575829/217011915-ca1989fb-10fc-4f57-89b7-6e5c60ada7fb.png">

<img width="175" alt="image" src="https://user-images.githubusercontent.com/6575829/217012037-5b567581-d73b-4c7f-b4c1-b66b1c81a2ea.png">

Windows' font rendering has been terrible since, well, forever and they never really did anything to improve it or bring it to same level as the Mac.

The root of why this is a problem in Cura is at least to my eyes, that you are using a font that is a) too thin and b) not optimised for screens. Probably something like Helvetica Thin. 

You should choose a font that is designed for screens and small sizes. Good examples that come to mind are:

Roboto - specifically designed for screens and small sizes. Default font for Google Material Design: https://github.com/googlefonts/roboto

IBM Plex - in my opinion a better font and used in the Carbon Design System by IBM: https://github.com/ibm/plex

**Comment by fieldOfView** [comment]
> The root of why this is a problem in Cura is at least to my eyes, that you are using a font that is a) too thin and b) not optimised for screens.

No, that is not it at all. The root is that the font is getting drawn at non-integer offsets.

**Comment by seltzdesign** [comment]
Okay, maybe its not the root, but it certainly doesn't help using such a thin font, which at the size it is being used means lines will always render at 1px or thinner.

Looking at software on Windows that I think has nice readable typography like Notion or Spotify, they all use fonts with more weight and more kerning - both of which increase readability. Even if the font was rendered perfectly, it would be too thin for optimum readability.

**Comment by d3dorsett** [comment]
I notice this is still an issue in 5.4.0. Oddly, in the menu, if you highlight the text it will correct itself until it gets scrolled up or down then if goes back to the pixelated text.
![image](https://github.com/Ultimaker/Cura/assets/50206276/8bc53000-a462-4b94-b879-9072edaf1e19)

**Comment by d3dorsett** [comment]
Oh wait. Oddly in Windows 11, I turned off "Smooth edges of images and text" and restarted Cura. It now looks correct. To verify, I then turned that feature back on and restarted Cura only to have it work correctly still. Weird.

![image](https://github.com/Ultimaker/Cura/assets/50206276/6b3d14e0-0d4d-40f0-a850-e7dff9bbb62d)

**Comment by h3rm** [comment]
> **Fix**?!? Oh wait. Oddly in Windows 11, I turned off "Smooth edges of images and text" and restarted Cura. It now looks correct. To verify, I then turned that feature back on and restarted Cura only to have it work correctly still. Weird.
> 
> ![image](https://user-images.githubusercontent.com/50206276/271131096-6b3d14e0-0d4d-40f0-a850-e7dff9bbb62d.png)

doesn't work for me in 5.5.0 beta1 :( https://github.com/Ultimaker/Cura/issues/17002#issuecomment-1766152481
